client/js/store/core/initStore.js

```diff
@@ -72,16 +72,22 @@ function initStore (storeModules, apiStoreModules, presetStoreModules) {
             },
             successCallbacks: [
               async (success) => {
-                const response = await success.json()
+                // If the response body is empty, contentType will be null
+                const contentType = success.headers.get('Content-Type')
```

Comment:
i can assume, that "success" can also be an already resolved json object..
Could we implement that case easily too? Or should we do that in a separat PR

Reply:
My main worry here is that I wouldn't be able to test the code, it might affect a lot of places in the application and we have code freeze today - so I'd prefer to do that in a separate PR.